SPECS/cloud-hypervisor-cvm/cloud-hypervisor-cvm.spec

```diff
@@ -0,0 +1,229 @@
+# This spec file assumes you're building on an environment where:
+# * You have access to the internet during the build
+# * You have rustup installed on your system
+# * You have both x86_64-unknown-linux-gnu and x86_64-unknown-linux-musl, or
+# * You have both aarch64-unknown-linux-gnu and aarch64-unknown-linux-musl
+#   targets installed.
+
+%define using_rustup 0
+%define using_musl_libc 0
+%define using_vendored_crates 1
+
+# Generated by update script
```

Comment:
We should remove this comment. Why do we need gitver, gitnum, gitrev variables? Those are only used once below.